### PR TITLE
Adjust documentation.

### PR DESCRIPTION
**Pull request by ingokuba**
- Listener instead of observer
- adjust table of content

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -39,7 +40,7 @@ Furthermore, this project uses a few open source image icons to make this applic
  The `Chat`, `Group` and `User` can be serialized to and deserialized from JSON. The file is stored at the project's root or from where the jar was executed.
 
 ### Design Patterns <a name="4"><a/>
- The `Chat` and `User` are observable objects. The client window object uses this observer pattern to react to changes in the objects. With this solution the UI and business logic can be separated easily without tightly coupling them. Furthermore multiple observers can be registered on an object. [3]
+ The `Chat` and `User` are observable objects, implemented with property change listeners. The client window object uses this observer pattern to react to changes in the objects. With this solution the UI and business logic can be separated easily without tightly coupling them. Furthermore multiple listeners can be registered on an object. [3]
```
Comment (thisIsTheFoxe) [inline DOCUMENTATION.md]: don't know if you need a PR for that..

File: DOCUMENTATION_de.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-# <div style="text-align: right">[:gb:](DOCUMENTATION.md)</div>
-
+<div style="text-align: right">[:gb:](DOCUMENTATION.md)</div>
```
Comment (thisIsTheFoxe) [inline DOCUMENTATION_de.md]: you should probably rename to `en`

File: DOCUMENTATION.md (hunk 1 above)
Comment (thisIsTheFoxe) [inline DOCUMENTATION.md]: is listener used as a name here.. in that case it should be Caps.

File: DOCUMENTATION.md [hunk 3]
```diff
@@ -5,15 +5,19 @@
...
 
 # Content
-### 1. [What it does](#1)
-### 2. [Design](#design)
+### 1. [What it does](#What-it-does)
+### 2. [Design](#Design)
 ### 3. [Implementation](#Implementation)
-### 4. [Testing](#testing)
-### 5. [Sources](#5)
+#### 		3.1 [Design Patterns](#Design-patterns)
```
Comment (thisIsTheFoxe) [inline DOCUMENTATION.md]: IMHO we shouldn't do that.. I mean it's not that each chapter has a bajillion paragraphs.. this makes it just unnecessary clumsy .-.

File: DOCUMENTATION.md (hunk 1 above)
Comment (ingokuba) [reply DOCUMENTATION.md]: Just to be sure. I don't know whether it's confusing for prof

Comment (ingokuba) [reply DOCUMENTATION.md]: hm, I had it like that - I will reword it

File: DOCUMENTATION.md [hunk 4]
```diff
@@ -1,18 +1,23 @@
...
-### 5. [Sources](#5)
+1. [What it does](#What-it-does)
+2. [Design](#Design)
+3. [Implementation](#Implementation)
+	1. [Design Patterns](#Design-patterns)
+4. [Testing](#Testing)
+	1. [Unit Tests](#Unit-tests)
+	2. [UI Tests](#UI-tests)
+	3. [Sonar](#Sonar)
+5. [Sources](#Sources)
```
Comment (ingokuba) [inline DOCUMENTATION.md]: What do you think? :thinking: better or worse?

File: DOCUMENTATION_de.md (hunk 2 above)
Comment (ingokuba) [reply DOCUMENTATION_de.md]: you mean like 'DOCUMENTATION_gb.md`?

Comment (ingokuba) [reply DOCUMENTATION_de.md]: or `DOCUMENTATION_en.md`

File: DOCUMENTATION.md (hunk 4 above)
Comment (thisIsTheFoxe) [reply DOCUMENTATION.md]: Sorry.. with my fam tdy.. interweb in the village is over 9000 bits/s -.-

File: DOCUMENTATION.md (hunk 3 above)
Comment (ingokuba) [reply DOCUMENTATION.md]: yee ok... we don't even need a table of content :sweat_smile: too little content

File: DOCUMENTATION_de.md (hunk 2 above)
Comment (thisIsTheFoxe) [reply DOCUMENTATION_de.md]: I did mean the flag.. but then again it doesn’t matter thaat much..

Comment (ingokuba) [reply DOCUMENTATION_de.md]: Imo english acts as a default

Comment (ingokuba) [reply DOCUMENTATION_de.md]: us flag for english? BIG VETO

**Comment by ingokuba** [comment]
Hehe merged

**Comment by thisIsTheFoxe** [comment]
just.. why? I think you should change the settings of this repo... 😤

**Pull request merged**